app/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,5 @@
     <string name="request_permission">This sample needs camera permission.</string>
     <string name="camera_error">This device doesn\'t support Camera2 API.</string>
     <string name="help_message">When the LED is on, press the button to capture an image</string>
+    <string name="take_photo">Take Photo</string>
```

Comment:
I think this is not needed anymore, right?

Reply:
Agreed, thanks!